### ISSUE TITLE
Bug with toggle radio overlaping fixed/absolute positioned elements

### ISSUE THREAD

**Issue by batata004** [Semantic-Org/Semantic-UI#5742]
Hi,

Please check the image below. I have a fixed nav link at the top of the screen however when I scroll down the toggle radio overlaps it. 

![aa](https://user-images.githubusercontent.com/8988923/30035618-80e362dc-9182-11e7-9d14-4915c6fa49f7.jpg)

I tried simply changing the z-index of the `input type="radio"` element  but it has no effect! I discovered the problem is not with the z-index of the input but with the z-index of the white circle. I debugged it with F12 and I have no idea how semantic ui transformed an input into that beautiful button.

I tried finding the class of the white circle to override it's z-index but I cant find the white circle to inspect in the DOM! Any suggestion how the user can override the default z-index of the white circle?

**Comment by Banandrew** [comment]
Hi @batata004, here it is:

![image](https://user-images.githubusercontent.com/6409354/30041277-69befde0-91f1-11e7-8446-27a226e0ccf8.png)

Although, changing its `z-index` is probably a bad idea, because it might break the checkbox’s component relationship with other SUI components, so it’d probably be best to increase the menu’s `z-index` instead.

`.ui.fixed.menu` uses a `z-index` of `101`, so it works with the checkbox—do you use a custom fixed menu?

**Comment by batata004** [comment]
@Banandrew  I use a custom fixed menu ( I did by myself) but inside of it I use only semantic ui components. You can see the bug happening here -> [https://www.sitepor500.com.br/?q=briefing-criacao-site](https://www.sitepor500.com.br/?q=briefing-criacao-site)

Thanks for pointing the ::after element, I was not aware semantic ui would style the ::after element which is a child inside the sibbling of the input itself! I tried changing style of the :;after element and it worked but the modal window (I have several in this website) didnt work well over it.

Anyway, I followed your suggestion and increase the z-index of the fixed menu and it worked BUT when I went to the /index.php (home) of the webiste some menus didnt appear right. So I had to fix many other z-indexes and now I think everything works fine.

I appreciate your help my friend, the problem was solved with your suggestion!

**Comment by batata004** [comment]
Actually now that I was answering you I had a much better idea that I just implemented! I gave the form (which is the parent of all the checkbox) a position:relative and z-index:0 and it worked perfectly! I just did that and restore alllll the other z-indexes of all other elements on page and it works fine!

So my advise to anyone facing this problem is to give a z-index to the parent element of the checkboxes instead of trying to play with the ::after style or anything else! Hope this helps someone!
